### PR TITLE
5 add example usages of utilities

### PR DESCRIPTION
**Pull request by robert-ervin-jones**
Adds remaining utilities

### CODE REVIEW

**Comment by atice** [comment]
Looks good overall. Below are some issues I found.

astral_runner.py 
1) move ` to right of <suffix> in optional arguments
2)Should /RAxML/ really be in the -i option in the **NOTE** section

build_database.py
1) replate "we recommend users rerun build_dataset.py to update profile HMMs, and blast databases to" with "will" in the **NOTE** section. This is an antiquated instruction since apply_to_db.py now automatically does this.

fast_site_remover.py
1) `dist_est` in the && section was a link to the dist_est documentation in the manual. Is it here. If not can you change it to be that same link?

fast_taxa_remover.py
1) Same as above only with PREQUAL and Divvier
2) Sometimes periods are used at the end of explanations and sometimes not. Look into homogenizing

heterotachy.py
1) Sometimes periods are used at the end of explanations and sometimes not. Look into homogenizing
2)`dist_est` in the && section was a link to the dist_est documentation in the manual. Is it here. If not can you change it to be that same link?

mammal_modeler.py
1) The last instance of MAMMaL was a link to the MAMMaL documentation in the manual. Is it here. If not can you change it to be that same link?

File: pages/utilites/astral_runner.md [hunk 1]
```diff
@@ -37,4 +37,4 @@ Default `astral_runner.py` output:
   - A greedy consensus of the 100 bootstrapped replicate trees.
   - The “main” ASTRAL tree.
 
-**NOTE**: Users may choose to use the trees generated by `sgt_constructor.py` using the ortholog files generated by `matrix_constructor.py`. For example, `sgt_constructor.py –trees_only -i matrix_constructor_out_<M.D.Y>/trimal/ -o <above, directory of your choice>/`. If users construct trees via this method, example run is `rtc_binner.py -i <directory of your choice>/RAxML/ -st gt80trimal.tre -s gt80trimal`.
\ No newline at end of file
+**NOTE**: Users may choose to use the trees generated by `sgt_constructor.py` using the ortholog files generated by `matrix_constructor.py`. For example, `sgt_constructor.py –trees_only -i matrix_constructor_out_<M.D.Y>/trimal/ -o <above, directory of your choice>/`. If users construct trees via this method, example run is `rtc_binner.py -i sgt_constructor_out_<M.D.Y>/trees/ -st gt80trimal.tre -s gt80trimal`.
```
Comment (atice) [inline pages/utilites/astral_runner.md:40]: Change "`rtc_binner.py -i sgt_constructor_out_<M.D.Y>/trees/ -st gt80trimal.tre -s gt80trimal`." to "astral_runner.py -i sgt_constructor_out_<M.D.Y>/trees/"

Comment (robert-ervin-jones) [reply pages/utilites/astral_runner.md:40]: Ah I see.

**Review by atice — APPROVED** [review]
Looks good.

**Pull request merged**